src/components/form/Input/styles.css

```diff
@@ -1,16 +1,12 @@
-.Input {
-  margin: 0.3rem 0 0.8rem;
-}
-
 .Input input {
-  border-radius: 5px;
-  border: 1px solid var(--color-black);
+  border: 1px solid var(--color-light-black);
```

Comment:
Consider installing Sass for the usage of variables and mixins.

Reply:
Ok, I already installed Sass